onnxruntime/core/framework/execution_frame.h

```diff
@@ -51,9 +51,12 @@ class IExecutionFrame {
   const OrtValue* GetNodeInputOrOutputMLValue(int index) const;
   OrtValue* GetMutableNodeInputOrOutputMLValue(int index);
 
-#ifdef ENABLE_TRAINING
+#if !defined(ORT_MINIMAL_BUILD) && !defined(DISABLE_CONTRIB_OPS)
```

Comment:
This should not be enabled by default in all inference builds. Either protect it around a separate macro called ENABLE_ATEN_FALLBACK.